js/main.js

```diff
@@ -259,6 +263,29 @@ class App extends React.Component {
     this._socket = socket;
   }
 
+  _checkWindow = (cmd, numTries) => {
+    if (cmd.win in this.state.consistent_pane_copy) {
+      let windowContent = this.state.consistent_pane_copy[cmd.win];
```

Comment:
Nit: `# Apply patch and check hash. Re-fetch if final window doesn't match hash`